src/Akka.Streams.Kafka/Dsl/KafkaConsumer.cs

```diff
@@ -42,7 +44,8 @@ public static class KafkaConsumer
         /// </summary>
         public static Source<ConsumeResult<K, V>, IControl> PlainExternalSource<K, V>(IActorRef consumer, IManualSubscription subscription)
         {
-            return Source.FromGraph(new ExternalPlainSourceStage<K, V>(consumer, subscription));
+            return Source.FromGraph(new ExternalPlainSourceStage<K, V>(consumer, subscription))
+                .AddAttributes(ActorAttributes.CreateSupervisionStrategy(new DefaultConsumerDecider<K, V>().Decide));
```

Comment:
Wrong place to do it. You do it here: https://github.com/akkadotnet/Akka.Streams.Kafka/blob/4f4228199fdacb83d772367c5328812cf9f65357/src/Akka.Streams.Kafka/Stages/Consumers/Abstract/KafkaSourceStage.cs#L42

Reply:
Fix this everywhere.

Reply:
This does not work, InitialAttributes is somehow injected to the end of the attribute list, not the first.